src/pyDataverse/docs/source/user/basic-usage.rst

```diff
@@ -361,6 +371,51 @@ Dataset) attached to it.
 
 Now the Dataverse instance is as it was once before we started.
 
+.. _user_basic-usage_download-data:
```

Comment:
I think this section should be earlier - after the "Upload Datafile" and before the "Retrieve all created data as a Dataverse tree" section.

Reply:
Ok changed!